board/drivers/gpio.h

```diff
@@ -10,6 +10,11 @@
 #define OUTPUT_TYPE_PUSH_PULL 0U
 #define OUTPUT_TYPE_OPEN_DRAIN 1U
 
+#define SPEED_LOW 0U
```

Comment:
maybe change to `GPIO_SPEED_x` for clarity since these will be globally in scope.  Are these speed grades (`LOW`, `MEDIUM`, etc) the same name as in the datasheet?

Reply:
Speed grades are the same and naming was taken from HAL lib. If you want to add GPIO_ in front - I would add it to all defines. What do you think?